examples/meander/meander_example.py

```diff
@@ -0,0 +1,157 @@
+            flux_constant.assign((gradient_flux*t_new*morfac) - 0.02)
+        else:
+            flux_constant.assign((gradient_flux2*(t_new*morfac-6000)) - 0.053)
+            elev_constant.assign(gradient_elev2*(t_new*morfac-18000) + elev_init_const)
+        t_old.assign(t_new)
+
+
+t_old = Constant(0.0)
+
+# define mesh
+mesh2d = Mesh("meander.msh")
+x, y = SpatialCoordinate(mesh2d)
+
+# define function spaces
+V = FunctionSpace(mesh2d, 'CG', 1)
+DG_2d = FunctionSpace(mesh2d, 'DG', 1)
+vector_dg = VectorFunctionSpace(mesh2d, 'DG', 1)
+
+# define underlying bathymetry
+
+bathymetry_2d = Function(V, name='Bathymetry')
+
+gradient = Constant(0.0035)
+
+L_function = Function(V).interpolate(conditional(x > 5, pi*4*((pi/2)-acos((x-5)/(sqrt((x-5)**2+(y-2.5)**2))))/pi, pi*4*((pi/2)-acos((-x+5)/(sqrt((x-5)**2+(y-2.5)**2))))/pi))
+bathymetry_2d1 = Function(V).interpolate(conditional(y > 2.5, conditional(x < 5, (L_function*gradient) + 9.97072, -(L_function*gradient) + 9.97072), 9.97072))
+
+init = max(bathymetry_2d1.dat.data[:])
+final = min(bathymetry_2d1.dat.data[:])
+
+bathymetry_2d2 = Function(V).interpolate(conditional(x <= 5, conditional(y <= 2.5, -9.97072 + gradient*abs(y - 2.5) + init, 0), conditional(y <= 2.5, -9.97072 - gradient*abs(y - 2.5) + final, 0)))
+bathymetry_2d = Function(V).interpolate(-bathymetry_2d1 - bathymetry_2d2)
+
+# record initial bathymetry before it evolves
+initial_bathymetry_2d = Function(V).interpolate(bathymetry_2d)
+
+# choose directory to output results
+ts = time.time()
+st = datetime.datetime.fromtimestamp(ts).strftime('%Y-%m-%d %H:%M:%S')
+outputdir = 'outputs' + st
+
+# initialise velocity and elevation
+chk = DumbCheckpoint("hydrodynamics_meander/elevation", mode=FILE_READ)
+elev = Function(DG_2d, name="elevation")
+chk.load(elev)
+chk.close()
+
+chk = DumbCheckpoint('hydrodynamics_meander/velocity', mode=FILE_READ)
+uv = Function(vector_dg, name="velocity")
+chk.load(uv)
+chk.close()
+
+morfac = 50
+dt = 2
+end_time = 5*3600
+
+if os.getenv('THETIS_REGRESSION_TEST') is not None:
+    # the example is being run as a test
+    # run the spin-up by importing it
+    import meander_hydro  # NOQA
+    end_time = 3600.
+
+viscosity_hydro = Constant(5*10**(-2))
+
+# set up solver
+solver_obj = solver2d.FlowSolver2d(mesh2d, bathymetry_2d)
+options = solver_obj.options
+
+# this test case only uses bedload transport but using all slope effect corrections and secondary current
+options.sediment_model_options.solve_suspended_sediment = False
+options.sediment_model_options.use_bedload = True
+options.sediment_model_options.solve_exner = True
+options.sediment_model_options.use_angle_correction = True
+options.sediment_model_options.use_slope_mag_correction = True
+options.sediment_model_options.use_secondary_current = True
+options.sediment_model_options.use_advective_velocity_correction = False
+options.sediment_model_options.morphological_viscosity = Constant(1e-6)
+
```

Comment:
skipping the blank lines would make the file shorter

Reply:
have removed most blank lines